cylc/rose/entry_points.py

```diff
@@ -226,6 +226,20 @@ def copy_config_file(
     opts=None,
     dir_=None,
 ):
+    """Copy the ``rose-suite.conf`` from a workflow source to run directory.
+
+    Args:
+        dir_ (pathlib.Path | or str):
+            Source Path of Cylc install.
+        opts:
+            Not used in this function, but requried for consistent entry point.            
+        dest_root (pathlib.Path | or str):
+            Destination path of Cylc install - the workflow rundir.
```

Comment:
Thanks, but order is backwards

Reply:
Fixed, but by changing the order of the kwargs. Bear in mind that I'll be refactoring this early next week. I think that the order should be "sourcedir", "opts", "rundir".